include/exadg/incompressible_navier_stokes/postprocessor/output_generator.cpp

```diff
@@ -452,11 +452,12 @@ OutputGenerator<dim, Number>::calculate_additional_fields(VectorType const & vel
 
     if(output_data.write_cfl)
     {
-      auto   param          = navier_stokes_operator->get_param();
-      double time_step_size = navier_stokes_operator->calculate_time_step_cfl(velocity);
-      time_step_size *= param.cfl;
+      // This time step size corresponds to CFL = 1.
+      auto time_step_size = navier_stokes_operator->calculate_time_step_cfl(velocity);
```

Comment:
```suggestion
      auto const time_step_size = navier_stokes_operator->calculate_time_step_cfl(velocity);
```

Reply:
The functions in "time_integration/time_step_calculation.h" document this. One could argue that this should be written redundantly in IncNS::SpatialOperatorBase ... this would be a fix of PR #105

Reply:
done